packages/theme-default/src/styles/vars.css

```diff
@@ -52,15 +52,15 @@
 }
 
 .dark {
-  --rp-c-bg: #23272f;
+  --rp-c-bg: #191D24;
   --rp-c-bg-soft: #292e37;
   --rp-c-bg-mute: #343a46;
   --rp-c-bg-alt: #000;
 
   --rp-c-divider: rgba(84, 84, 84, 0.65);
   --rp-c-divider-light: rgba(84, 84, 84, 0.48);
 
-  --rp-c-text-1: rgba(255, 255, 255, 0.87);
+  --rp-c-text-1: rgba(255, 255, 255, 1);
```

Comment:
It looks too bright, how about using`rgba(255, 255, 245, 0.93)`?

Reply:
The contrast ratio will be 14.6. Looks good to me.

![CleanShot 2024-10-08 at 15 15 13@2x](https://github.com/user-attachments/assets/95959856-0cce-43de-a087-fd017b4a3be2)